<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QWidget" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>789</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ИС Поликлиниа</string>
  </property>
  <widget class="QTabWidget" name="tab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>431</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_record">
    <attribute name="title">
     <string>Запись на процедуры</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>746</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_proc">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Процедура</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_proc">
          <property name="minimumSize">
           <size>
            <width>340</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_cab">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Кабинет</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_cabinet">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_day">
          <property name="text">
           <string>День</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_day">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_show">
          <property name="minimumSize">
           <size>
            <width>115</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Просмотр</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tabel_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>741</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>741</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_record">
        <property name="text">
         <string>Записаться</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_show_record">
    <attribute name="title">
     <string>Активные записи</string>
    </attribute>
    <widget class="QTableWidget" name="table_activ_proc">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delete">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>360</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_reselt">
    <attribute name="title">
     <string>Результат процедур</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>711</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Тут можно отображать результаты уже выполненных анализов, в варианте не прописано</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_logout">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>470</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_patient">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Пациент:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="patient_name">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Бесплатные процедуры:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="free_proc">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
